### PR TITLE
Writing of system metadata for schedules created via new APIs.

### PR DESCRIPTION
**Pull request by anwar6953**
https://builds.cask.co/browse/CDAP-RUT1035

### CODE REVIEW

File: cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/deploy/pipeline/SystemMetadataWriterStageTest.java [hunk 1]
```diff
@@ -81,6 +81,9 @@ public void testWorkflowTags() throws Exception {
     Sets.SetView<String> intersection = Sets.intersection(workflowSystemTags, getWorkflowForkNodes(workflowSpec));
     Assert.assertTrue("Workflows should not be tagged with fork node names, but found the following fork nodes " +
                         "in the workflow's system tags: " + intersection, intersection.isEmpty());
+
+    Map<String, String> metadataProperties = metadataStore.getMetadata(MetadataScope.SYSTEM, appId).getProperties();
+    Assert.assertEquals("testSchedule:testDescription", metadataProperties.get("schedule:testSchedule"));
```
Comment (ecfm) [inline cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/deploy/pipeline/SystemMetadataWriterStageTest.java]: Better to use some static String for constructing this

Comment (anwar6953) [reply cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/deploy/pipeline/SystemMetadataWriterStageTest.java]: are you talking about the string on the left or the string on the right?

Comment (ecfm) [reply cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/deploy/pipeline/SystemMetadataWriterStageTest.java]: a SCHEDULE_NAME String can be used in both

Comment (anwar6953) [reply cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/deploy/pipeline/SystemMetadataWriterStageTest.java]: done

**Comment by ecfm** [comment]
LGTM

**Pull request merged**